libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/AdaptiveDialog.cs

```diff
@@ -406,6 +464,74 @@ protected async Task<bool> ProcessEventAsync(SequenceContext sequenceContext, Di
             return handled;
         }
 
+        protected async Task<bool> ProcessFormAsync(SequenceContext sequenceContext, CancellationToken cancellationToken)
+        {
+            DialogEvent evt;
+            var queues = EventQueues.Read(sequenceContext);
```

Comment:
>var queues = EventQueues.Read(sequenceContext); [](start = 12, length = 47)

can you add comments to this section of code describing the logic of what you are doing?